Server/Game/routeunithelper.py

```diff
@@ -58,12 +58,12 @@ def setTeam(data):
 	elif not "Perks" in data:
 		error = "The perk information is incomplete."
 	# Ensure the user selected the proper number of units - too many, cheater!
-	elif len(data["Units"]) > version.unit_count:
-		error = "Too many units have been selected (" + str(len(data["Units"])) + ")."
+	elif len(data["Units"]) > version.unit_max:
+		error = "Too many units have been selected ({0}).".format(len(data["Units"]))
 	# Ensure the user selected the proper number of units - too few, oops, add a few!
-	elif len(data["Units"]) < version.unit_count:
-		error = ("You must select " + str(version.unit_count) + " units, only "
-			+ str(len(data["Units"])) + " chosen.")
+	elif len(data["Units"]) < version.unit_min:
+		error = ("You must select at least {0} unit(s), {1} chosen.").format(version.unit_min,
+			"none" if (len(data["Units"]) == 0) else len(data["Units"]))
```

Comment:
I see what you are doing here but why starting and ending parenthesis?  It seems like this may end up as a string but based on this line it looks like it will resolve to a set with a single string element.